### PR TITLE
nginx: Enabled operation_name to be specifed inside a server block

### PR DESCRIPTION
**Pull request by tobiasstadler**

### CODE REVIEW

File: instrumentation/nginx/src/script.h [hunk 1]
```diff
@@ -26,6 +26,10 @@ struct NgxCompiledScript {
 
     return true;
   }
+
+  bool isValid() {
```
Comment (seemk) [inline instrumentation/nginx/src/script.h:30]: Perhaps nitpicking, but maybe `isEmpty`? `isValid` has more of a "is in an error state" vibe :smile:

Comment (tobiasstadler) [reply instrumentation/nginx/src/script.h]: Yes, `IsEmpty` sounds better

**Review by seemk — APPROVED** [review]
Thanks :+1:

**Comment by tobiasstadler** [comment]
Thank You!

**Pull request merged**